package.json

```diff
@@ -31,24 +31,25 @@
     "ember-cli-htmlbars-inline-precompile": "^1.0.2",
     "ember-cli-inject-live-reload": "^1.7.0",
     "ember-cli-pretender": "1.0.1",
-    "ember-cli-qunit": "^4.3.0",
+    "ember-cli-qunit": "^4.3.1",
```

Comment:
Updating to 4.3.2 creates an error:
"calling set on destroyed object: <dummy@component:number-field::ember1442>.isFocused = true"
in test
"Integration | Component | {{number-field}}: arrow up from negative to positive"